css/css-text/line-breaking/line-breaking-012.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<html>
+<meta charset="utf-8">
+<title>CSS Text — line breaking for nowrap and floats</title>
+<meta name=assert content="When whitespace: nowrap is specified, floats should not cause line breaks">
+<link rel=help href="https://www.w3.org/TR/css-text-3/#white-space-property">
+<link rel=match href="reference/line-breaking-002-ref.html">
```

Comment:
The current set of line-breaking tests seem to be setup so that they all match the same reference file (css/css-text/line-breaking/reference/line-breaking-001-ref.html). This test needs a different reference as 001-ref.html tests when a line break *does* occur whilst this checks that one doesn't occur. So 002-ref.html :).

Reply:
It is common practice to number the ref after the first test that matches it, so this would be -012-ref, even though there's not -002 through -011. Not a blocker though.

Reply:
Ack, moved to -012-ref